include/libcyphal/transport/svc_sessions.hpp

```diff
@@ -94,6 +135,10 @@ class IResponseTxSession : public ITxSession
     ///
     virtual cetl::optional<AnyError> send(const ServiceTransferMetadata& metadata,
```

Comment:
Unrelated to these changes, but who calls `send()`?

Reply:
Either user (if she works with so low "transport" level of libcyphal), or upper "presentation" layer (which is the future work after transport layer completion). OR, unit tests of cause ;)